File: trunk/testdata/test_code.cc
// Copyright 2007 Google Inc. All Rights Reserved.
// Author: nigdsouza@google.com (Nigel D'souza)
//
// A sample .cc file for local_gentags testing.

#include "tools/tags/logging.h"
#include "tools/tags/tagsoptionparser.h"

void HelloWorld(const char* name) {
  LOG(INFO) << "Hello World, " << name << '\n';
}

int main(int argc, char **argv) {
  ParseArgs(argc, argv);
  HelloWorld(argv[1]);
}

File: testdata/test_code.py
#!/usr/bin/python2.4
#
# Copyright 2007 Google Inc. All Rights Reserved.

"""A sample .py file for local_gentags testing."""

__author__ = "nigdsouza@google.com (Nigel D'souza)"

import sys


def HelloWorld(name):
  """Prints 'Hello World' with the given name."""
  print 'Hello World, %s!' % name

if __name__ == '__main__':
  HelloWorld(sys.argv[1])
